CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# v119.0 (In progress)
+# v119.0 (_2023-09-25_)
 
 [Full Changelog](In progress)
```

Comment:
This needs updating also. See https://github.com/mozilla/application-services/commit/0661b721c537f74ab10ecb6b87359f1ef3f55b6a for example.